index.bs

```diff
@@ -551,6 +559,59 @@ It must act as follows:
   </pre>
 </div>
 
+<h4 id="stream-method-algo">
+The stream method</h4>
+
+The <dfn method for=Blob>stream()</dfn> method returns the contents
+of the blob as a {{ReadableStream}}.
```

Comment:
Do we need this summary? It seems clearer if the `<dfn>` is immediately about the normative aspect.

Reply:
Good point. This sentence really doesn't add much, and if we want any kind of non-normative bits those should be domintro notes. Removed it here and for the other methods.